### PR TITLE
Add Data Landing Zone Subs as Input

### PR DESCRIPTION
**Pull request by marvinbuss**
## Summary of the Pull Request
This PR implements the necessary changes to support the changes here: https://github.com/Azure/data-landing-zone/pull/223

## References
See here https://github.com/Azure/data-landing-zone/pull/223

## PR Checklist
* [x] Closes Issue referenced here: https://github.com/Azure/data-landing-zone/pull/223
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA

## Validation Steps Performed
See comments and runs below

### CODE REVIEW

**Review by marvinbuss — COMMENTED** [review]
LGTM!

File: .github/linters/.arm-ttk.psd1 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @{
     # Test = @( )
     Skip = @(
-        'Template Should Not Contain Blanks',
+        'Template Should Not Contain Blanks'
```
Comment (amanjeetsingh) [inline .github/linters/.arm-ttk.psd1:7]: Change "Should" to "Must"

**Review by amanjeetsingh — COMMENTED** [review]
Minor change proposed. Other than, all good to go.

File: .github/linters/.arm-ttk.psd1 (hunk 1 above)
Comment (marvinbuss) [reply .github/linters/.arm-ttk.psd1:7]: This is pre defined. This cannot be changed.

**Review by amanjeetsingh — APPROVED** [review]

**Pull request merged**